modules/setup-dev-environment/pages/logging.adoc

```diff
@@ -46,6 +46,35 @@ Here is an example of the Log4j2 configuration for the logger used in the exampl
 <Logger name="org.bonitasoft.groovy.script" level="TRACE"/>
 ----
 
+
+==== Stream logs to a remote server
+
+You need to declare a socket in `log4j-appenders.xml` with a `JSONLayout` to stream to a remote server.
```

Comment:
Note that the SocketAppender and the JSONLayout are 2 distinct concept. You may use Socket with plain text logs.

Reply:
I would split in 2 distinct paragraphs. One about how to use the JSONLayout (which we already have in default configuration but is not used in Loggers). Another about the Socket for streaming.

Alternatively, we can also just update the log4j2-appenders.xml to add the Socket example appender. Make the appenders comments more self-explanatory.
And document here that we can add Loggers to use the appender we want.